src/main/java/net/minecraftforge/common/extensions/IForgeBlockState.java

```diff
@@ -749,4 +749,28 @@ default boolean canBeHydrated(BlockGetter getter, BlockPos pos, FluidState fluid
     {
         return self().getBlock().canBeHydrated(self(), getter, pos, fluid, fluidPos);
     }
+
+    /**
+     * Returns the factor by which movement speed is multiplied for an entity standing on this block.
```

Comment:
Documentation does not match block.

Reply:
What do you mean?

Reply:
The documentation is different from that provided on the delegated method.